api/schema.capnp

```diff
@@ -34,8 +34,27 @@ struct JobInfo {
     # that the server crashed while building, and when it came back up we
     # no longer wanted to test that commit anyway.
   }
+
+  queuedAt  :union {
+    ts         @6 :Float64;
+    # timestamp as seconds since epoch
+    none       @7 :Void;
```

Comment:
These numbers are unique from `@0` based on the `:union` see https://capnproto.org/language.html